frontend/server/src/Controllers/Course.php

```diff
@@ -696,7 +783,7 @@ private static function createAssignment(
      * @param int $problemsetId
      * @param int $userId
      * @param bool $validateVisibility validations no needed when it is a clone
-     * @param ?int $points = 100
+     * @param float|int|null $points = 100
```

Comment:
nit:

```suggestion
     * @param float|null $points
```

(o mejor aún, quitar esto y dejar L708 como estaba).

Reply:
Listo